content/en/docs/reference/command-line-tools-reference/feature-gates-removed.md

```diff
@@ -32,6 +32,9 @@ In the following table:
 |---------|---------|-------|-------|-------|
 | `Accelerators` | `false` | Alpha | 1.6 | 1.10 |
 | `Accelerators` | - | Deprecated | 1.11 | 1.11 |
+| `AdvancedAuditing` | `false` | Alpha | 1.7 | 1.7 |
+| `AdvancedAuditing` | `true` | Beta | 1.8 | 1.11 |
+| `AdvancedAuditing` | `true` | GA | 1.12 | 1.28 |
```

Comment:
```suggestion
| `AdvancedAuditing` | `true` | GA | 1.12 | 1.27 |
```

Reply:
Thanks @tengqm for looking at this and pointing out this error. By mistake, I wrote it to v1.27.
I will update it.

Reply:
@tengqm, I have addressed the above comment. PTAL!